eras/dijkstra/impl/src/Cardano/Ledger/Dijkstra/TxCert.hs

```diff
@@ -223,6 +222,17 @@ instance Era era => EncCBOR (DijkstraTxCert era) where
     DijkstraTxCertPool poolCert -> encodePoolCert poolCert
     DijkstraTxCertGov govCert -> encCBOR govCert
 
+dijkstraTotalRefundsTxCerts ::
```

Comment:
I think it is worth adding a comment about why this change is safe when comparing to previous eras
```suggestion
-- | Unlike previous eras, we no longer need to lookup refunds from the ledger state, since all of the certificates specify the actual refund and ledger rules will validate that they are accurate.
dijkstraTotalRefundsTxCerts ::
```